chapter2/tests/t20/terraform/base/remote_exec.tf

```diff
@@ -0,0 +1,25 @@
+data "template_file" "remote_exec" {
+  template = file("${path.cwd}/terraform/base/scripts/remote_exec.sh")
+  vars = {
+    name      = "Camp"
```

Comment:
I think `vars` can be moved to variables

Reply:
> I think `vars` can be moved to variables

@kseniyashaydurova 
I'm not sure about this, do you mean in variables.tf, or in block locals?

Reply:
I think vars with defaults can be set

Reply:
> I think vars with defaults can be set

@kseniyashaydurova 
Of course! Done